generators/base/generator-base.mjs

```diff
@@ -2500,6 +2505,77 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
       (dest.applicationType === MICROSERVICE && !dest.skipUserManagement);
 
     dest.generateBuiltInAuthorityEntity = dest.generateBuiltInUserEntity && !dest.databaseTypeCassandra;
+
+    if (dest.databaseTypeSql) {
+      const devDatabaseData = getDatabaseData(dest.devDatabaseType);
+      const prodDatabaseData = getDatabaseData(dest.prodDatabaseType);
```

Comment:
splitting technologies assignment (db, cache, ...) in dedicated method would be nice.
splitting assignment in dedicated methods too (jdbcDriver vs HibernateDialect vs...): it will for example easy to manage variability (i.e. no jpadialect for reactive)

Reply:
r2dbc, jdbc and hibernate are sql generator.
I don't see it will improve splitting them.